hal/armv7m/imxrt/117x/hal.c

```diff
@@ -88,8 +90,23 @@ addr_t hal_kernelGetAddress(addr_t addr)
 }
 
 
+int hal_memAddMap(addr_t start, addr_t end, u32 attr, u32 id)
+{
+	return mpu_mapAlloc(start, end, attr, id);
+}
+
```

Comment:
Body of `mpu_mapAlloc()` should be placed inside this function.

Reply:
`mpu_mapAlloc()` uses internal functions from `mpu.c`.
If the body of `mpu_mapAlloc()` is moved to `hal.c`, then probably the majority of the code from `mpu.c` will have to be moved to hal.c. Moreover, the code would be duplicated in `imxrt117x` and `imxrt106x`.
What do you think to keep functions related to `mpu` in one place and leave it as-is?

Reply:
@HBuczynski , that's right.